ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -132,6 +132,10 @@ func clusterUpFlags() []cli.Flag {
 			Name:  flags.ForceFlag + ", f",
 			Usage: "[Optional] Forces the recreation of any existing resources that match your current configuration. This option is useful for cleaning up stale resources from previous failed attempts.",
 		},
+		cli.StringFlag{
+			Name:  flags.ResourceTagsFlag,
+			Usage: "[Optional] Specify tags which will be added to AWS Resources created for your cluster. Specify in the format 'key1=value1,key2=value2,key3=value3'",
```

Comment:
I think it would be a good idea to list which EC2/VPC resources we're actually tagging, but the list is probably too verbose to include here. How about we add a note re: the `--tags` flag in the `README` which includes that list? (i.e., VPC, Subnets, Internet Gateway, Route Table, Security Group(s), & ASG)